language/move-prover/doc/user/spec-lang.md

```diff
@@ -3,18 +3,27 @@
 [PRE_POST_REFERENCE]: https://en.wikipedia.org/wiki/Design_by_contract
 
 
-# The Move Specification Language 1.0
+# Move Specification Language
 
-The *Move specification language*, abbreviated **MSL**, enables formal specification of Move programs.
-It plays together with the [Move Prover][PROVER], a tool which can verify the
-correctness of MSL specifications against Move code. In contrast to traditional testing, verification of MSL
-is exhaustive and holds for all possible inputs and global states of a Move module or transaction script.
-At the same time, it is fast and automated enough that it can be used at a similar place in the developer workflow
-where tests are, for example, for qualification of pull requests in continuous integration.
+The *Move specification language*, abbreviated **MSL**, is a subset of the Move language which enables formal
```

Comment:
Thanks -- describing this as a subset of Move seems like a step in the right direction. We might want to evolve the terminology in the future, especially to have a better term for the non-specification parts of Move. The "program expressions" used below is fine and I can't think of anything better right now, but perhaps we can come up with something better later.